.vscode/extensions.json

```diff
@@ -2,6 +2,5 @@
 	// See http://go.microsoft.com/fwlink/?LinkId=827846
 	// for the documentation about the extensions.json format
 	"recommendations": [
-		"eg2.tslint"
```

Comment:
use eslint?

Reply:
`eg2.tslint`  has been deprecated in favor of the `ms-vscode.vscode-typescript-tslint-plugin`

Reply:
This is to remove the workspace recommendation.